server/data/dataseed/get_authorities_aggregated_entry.js

```diff
@@ -0,0 +1,60 @@
+const _ = require('builders/utils')
+const { resolveEntrySeedsByExternalIds } = require('controllers/entities/lib/resolver/resolve_by_external_ids')
+const { isNotEmpty, objLength } = require('lib/utils/base')
+
+const authorities = {
+  bnb: require('data/bnb/get_bnb_entry_from_isbn'),
+  bne: require('data/bne/get_bne_entry_from_isbn'),
+  bnf: require('data/bnf/get_bnf_entry_from_isbn'),
+  wikidata: require('data/wikidata/get_wikidata_entry_from_isbn'),
+}
+
+const authoritiesNames = Object.keys(authorities)
+
+module.exports = async isbn => {
+  const entries = await Promise.all(authoritiesNames.map(wrap(isbn)))
+  // TODO: aggregate resolved data, rather than just returning the best one
```

Comment:
would be nice to actually to this todo before merge. We might miss some great data because of this ?
do you mind if i try @maxlath ?

Reply:
@jum-s you're welcome to try, but if the attempt isn't that succesful, I would prefer not having it blocking this PR as that should already be in improvement compared to the current situation

Reply:
i know, but all that PR to discard claims not found in the best authority is quite sad

Reply:
done with `5a7cf9ea` (branch has been rebased too)